defaults/main.yml

```diff
@@ -24,12 +24,29 @@ php_enable_php_fpm: false
 php_fpm_state: started
 php_fpm_handler_state: restarted
 php_fpm_enabled_on_boot: true
-php_fpm_listen: "127.0.0.1:9000"
-php_fpm_listen_allowed_clients: "127.0.0.1"
-php_fpm_pm_max_children: 50
-php_fpm_pm_start_servers: 5
-php_fpm_pm_min_spare_servers: 5
-php_fpm_pm_max_spare_servers: 5
+
+# PHP-FPM pool configuration.
+php_fpm_pools:
+  - pool_name: www
+    pool_template: www.conf.j2
+    pool_listen: "127.0.0.1:9000"
+    pool_listen_allowed_clients: "127.0.0.1"
+    pool_pm: dynamic
+    pool_pm_max_children: 50
+    pool_pm_start_servers: 5
+    pool_pm_min_spare_servers: 5
+    pool_pm_max_spare_servers: 5
+    pool_pm_max_requests: 500
```

Comment:
This is a new option—do you know if `500` is the default for max_requests?

Reply:
Default value is 0, which means "forever" according to documentation.

Here is a snippet from a fresh install on debian buster, from www.conf file:

```
; The number of requests each child process should execute before respawning.
; This can be useful to work around memory leaks in 3rd party libraries. For
; endless request processing specify '0'. Equivalent to PHP_FCGI_MAX_REQUESTS.
; Default Value: 0
;pm.max_requests = 500
```

Personally I'd set this value > 0, for example 2000.